### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by kanvuduc** [rjocoleman/atom-codebug#4]
Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:267:11)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:238:14)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:774:25)
Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:219:20)
```

**Comment by rjocoleman** [comment]
@kanvuduc I'm not using Atom right now, I'll happily merge a PR and cut a new release though!

**Comment by rjocoleman** [comment]
closing old issue - fixed in d77455f8f4384feac32dd8ba4840cc08e31c5601
